### PR TITLE
Add Tables release details for October

### PR DESCRIPTION
**Pull request by christothes**

### CODE REVIEW

File: releases/2020-10/dotnet.md [hunk 1]
```diff
@@ -27,6 +28,7 @@ To install any of our packages, please search for them via `Manage NuGet Package
 ```bash
 $> dotnet add package Azure.Messaging.EventHubs --version 5.3.0-beta.3
 $> dotnet add package Azure.Messaging.EventHubs.Processor --version 5.3.0-beta.3
+$> dotnet add package Azure.Data.Tables --version 3.0.0-beta.2
```
Comment (kinelski) [inline releases/2020-10/dotnet.md:31]: Could you move `Tables` above `EventHubs`? We should follow alphabetical order.

File: releases/2020-10/dotnet.md [hunk 2]
```diff
@@ -51,6 +53,16 @@ Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here
 
 - `EventData` has been integrated with the new Schema Registry service, via use of the `SchemaRegistryAvroObjectSerializer` with the `BodyAsBinaryData` member.
 
+### Azure.Data.Tables [Changelog](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/tables/Azure.Data.Tables/CHANGELOG.md#300-beta2-2020-10-06)
```
Comment (kinelski) [inline releases/2020-10/dotnet.md:56]: Same here. Alphabetical order.

File: releases/2020-10/dotnet.md [hunk 3]
```diff
@@ -51,6 +53,16 @@ Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here
...
+
+### Added
+
+- Implemented batch operations.
+
+### Changed
+
+- `TableClient`'s `GetEntity` method now exposes the `select` query option to allow for more efficient existence checks for a table entity.
+
```
Comment (kinelski) [inline releases/2020-10/dotnet.md:65]: We should follow the naming specified in the [Release Notes Policies](https://azure.github.io/azure-sdk/policies_releasenotes.html#whats-a-developer-impacting-change). `Added` => `New Features`.

For `Changed`, it depends (sounds like a new feature to me), but if it's breaking, it should go in `Breaking Changes`.

File: releases/2020-10/dotnet.md [hunk 4]
```diff
@@ -51,6 +53,16 @@ Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here
 
 - `EventData` has been integrated with the new Schema Registry service, via use of the `SchemaRegistryAvroObjectSerializer` with the `BodyAsBinaryData` member.
 
+### Azure.Data.Tables [Changelog](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/tables/Azure.Data.Tables/CHANGELOG.md#300-beta2-2020-10-06)
+
+### Added
+
+- Implemented batch operations.
+
+### Changed
```
Comment (kinelski) [inline releases/2020-10/dotnet.md:62]: Could you reduce the header size for those (`Added`, `Changed`, `New Features`, ...). Usually, we use `####` for those.

**Review by kinelski — CHANGES_REQUESTED** [review]

**Review by kinelski — APPROVED** [review]
LGTM.

**Comment by christothes** [comment]
check-enforcer /override

**Pull request merged**